tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2537,5 +2594,136 @@ Status StorageManagerCanonical::group_metadata_vacuum(
   return consolidator->vacuum(group_name);
 }
 
+void StorageManager::array_get_fragment_tile_max_size(
```

Comment:
Can we move this code to another class than storage manager? Perhaps a new class?

Reply:
I think a new class is probably a bad idea, unless there are other (related? unrelated?) items anticipated as being usefully added to that same class (class KitchenSink? or GarbageCan? EsotericEntities? ah, MiscHelpers?).

Possibly as a static method of core Array? 
Although it would be sort of an outlier added in that fashion...

Reply:
> it would be sort of an outlier

Not for very long. Paul and I have been planning on how to move the operations on arrays to be member functions of `class Array`. This is in preparation for a major rework of that class to distinguish between "an array in storage" and "an array we're using right now (i.e. open)".

Reply:
So @eric-hughes-tiledb  you would approve of having it as a static method of the Array class?

Another possibility might be to somehow make it a part of the FragmentInfo class...